packages/kamino-sdk/src/Kamino.ts

```diff
@@ -3842,6 +3842,30 @@ export class Kamino {
     );
   };
 
+  processRebalanceParams = async (
+    dex: Dex,
+    pool: PublicKey | WhirlpoolWithAddress,
+    rebalanceType: Decimal,
+    rebalanceParams: Decimal[]
+  ): Promise<Decimal[]> => {
+    let processedRebalanceParams = [...rebalanceParams];
+    let rebalanceTypeKind = numberToRebalanceType(rebalanceType.toNumber());
+    if (dex == 'ORCA') {
+      const { address, whirlpool: whilrpoolState } = await this.getWhirlpoolStateIfNotFetched(pool);
```

Comment:
When is this endpoint used?

Reply:
currently we take the tick from the UI for `drift` so it gets called both on `buildStrategy` and on `updateStrategy`